native_client/Makefile

```diff
@@ -16,32 +16,32 @@ include definitions.mk
 default: $(DEEPSPEECH_BIN)
```

Comment:
maybe we should rename `DEEPSPEECH_BIN` now?

Reply:
I'm leaving implementation stuff to be renamed later to avoid making this even more hairy.

Reply:
ok, it's just that you already went through the trouble of adding missing `DEEPSPEECH_BIN` :)